tests/smtp-long-DATA-line-02-frames/test.rules

```diff
@@ -0,0 +1,6 @@
+alert smtp any any -> any any (frame:smtp.command_line; content:"MAIL|20|FROM:"; startswith; content:"xxxxx.co.uk"; distance:0; content:">|0d 0a|"; endswith; sid:1;)
+alert smtp any any -> any any (frame:smtp.data; content:"Reply-To:"; startswith; content:"Subject"; distance:0; content:"This is a multi-part message in MIME format."; distance:0; sid:2;)
+alert smtp any any -> any any (frame:smtp.response_line; content:"220 smtp001.mail.xxx.xxxxx.com ESMTP"; startswith; sid:3;)
+alert smtp any any -> any any (frame:smtp.stream; content:"|0d 0a|.|0d 0a|"; sid:4;)
```

Comment:
Why do not I see this one in `SMTPFrameTypes` ?

Reply:
it's a built in frame that is created for each (tcp) app-layer

Reply:
Is it documented ? (I cannot find it)
Where does it start ? Where does it end ?

Reply:
Starts at start of stream, ends at end of stream.

It's essentially a frame that is the entire stream, but exposed as <appproto>.stream. Behaves like:
```
alert <app proto> ... flow:only_stream; content:"abc"; 
```

Reply:
Thanks for the explanation